src/Promitor.Tests.Unit/Serialization/v1/Providers/LogAnalyticsDeserializerTests.cs

```diff
@@ -0,0 +1,43 @@
+using Promitor.Core.Scraping.Configuration.Serialization;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Model;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Model.ResourceTypes;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Providers;
+using Xunit;
+
+namespace Promitor.Tests.Unit.Serialization.v1.Providers
+{
+    public class LogAnalyticsDeserializerTests : ResourceDeserializerTest<LogAnalyticsDeserializer>
+    {
+        private readonly LogAnalyticsDeserializer _deserializer;
+
+        public LogAnalyticsDeserializerTests()
+        {
+            _deserializer = new LogAnalyticsDeserializer(Logger);
+        }
+
+        [Fact]
+        public void Deserialize_AccountNameSupplied_SetsAccountName()
+        {
+            const string workspaceId = "123-456-789";
+            YamlAssert.PropertySet<LogAnalyticsResourceV1, AzureResourceDefinitionV1, string>(
+                _deserializer,
+                $"workspaceId: {workspaceId}",
+                workspaceId,
+                r => r.WorkspaceId);
+        }
+
+        [Fact]
+        public void Deserialize_AccountNameNotSupplied_Null()
```

Comment:
AFAIK we also support subscription ID on this level, can you verify and add a test for it as well please?

Also, can you update the test name please?